client/src/components/participant-details/track-graduation.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Necessary?

Reply:
General purpose eslint-disable should *never* make it into master. This was probably just left over from testing.

Reply:
It's probably left over from testing. Should not make it into master, though.